tests/test_client.py

```diff
@@ -1,12 +1,15 @@
-import openreview
+from openreview import *
```

Comment:
why change this? you're using the openreview namespace below anyway, which is good practice (e.g. line 8)

is it so that you can avoid doing `openreview.openreview.OpenReviewException` on line 31? If so, we should fix the package structure.